ocr/utils.py

```diff
@@ -0,0 +1,48 @@
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def resize_aspect_ratio(gray_img, max_shape=1500):
+    """
+    Resize the image given the biggest size of any axis mantaining the aspect rate
+    """
+    x_shape, y_shape = gray_img.shape
+    if x_shape >= y_shape:
+        y_shape = int((1 - ((x_shape - max_shape) / x_shape)) * y_shape)
+        x_shape = max_shape
+    else:
+        x_shape = int((1 - ((y_shape - max_shape) / y_shape)) * x_shape)
+        y_shape = max_shape
+
+    resized_img = cv2.resize(gray_img
+                             , (y_shape, x_shape)
+                             , interpolation=cv2.INTER_CUBIC)
+    return resized_img
+
+
+def remove_shades(image):
```

Comment:
I found this method that looks interesting: https://arxiv.org/pdf/1710.05073.pdf. Could you check it and see whether it's worth it to add this link in a comment here?

Reply:
yeah looks promising, i will comment it